hypersistence-utils-hibernate-60/src/main/java/io/hypersistence/utils/hibernate/id/BatchSequence.java

```diff
@@ -0,0 +1,51 @@
+package io.hypersistence.utils.hibernate.id;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import org.hibernate.annotations.IdGeneratorType;
+
+/**
+ * Meta annotation to use {@link BatchSequenceGenerator} as an identifier generator.
+ * 
+ * @author Philippe Marschall
+ * @since 3.7.8
+ */
+@IdGeneratorType(BatchSequenceGenerator.class)
+@Retention(RUNTIME)
+@Target({ FIELD, METHOD })
+public @interface BatchSequence {
+
+    /**
+     * Returns the name of the sequence to use.
+     * 
+     * @return the name of the sequence to use
+     */
+    String name();
```

Comment:
An argument can be made this should be named `value` instead. That way the when going with the default fetch size the code is a little bit more concise

```java
@BatchSequence("SEQ_MY_ENTITY")
``` 
However when going with a custom fetch size it is a bit more awkward

```java
@BatchSequence(value = "SEQ_MY_ENTITY", fetchSize = 100)
``` 

let me know what you think.